launch_parser_on_nidm.py

```diff
@@ -35,6 +35,9 @@ def main():
 
     opt = parser.parse_args()
 
+    opt.input_dir == "examples/from_parsers/spm"
+    opt.output_dir == "examples/from_parsers"
+
```

Comment:
Are these 2 lines necessary ? Since they do not modify the behavior of  _[launch_parser_on_nidm.py](https://github.com/bids-standard/BEP028_BIDSprov/pull/102/files#diff-7ae14dacd3ee09443d5494fd1bc9340a965de9ea4bc94deaf8e3e2bb7bdb6b81)_